docs/development.md

```diff
@@ -0,0 +1,108 @@
+# Development
+## Running locally
+
+### Major Dependencies
+
+- [Node.js](https://nodejs.org/en/) (>= 14)
+- [NPM](https://www.npmjs.com/) (>= 6.14)
+- [Redis](https://redis.io/) (>= 5.0.5)
+
+### Installation
+
+1. Clone repository and change directory:
+
+   ```
+   git clone https://github.com/ministryofjustice/hmpps-book-secure-move-frontend && cd hmpps-book-secure-move-frontend
+   ```
+
+1. Install node dependencies:
+
+   ```
+   npm install
+   ```
+
+1. Build the assets
+
+    ```
+    npm run build
+    ```
+
+## Configuring the application
+
+Create a copy of the example environment variable file and add values for the keys:
+
+   ```
+   cp .env.example .env
+   ```
+
+Set the [environment variables](#environment-variables) accordingly.
+
+## Running the application
+
+### In production mode
+
+```
+NODE_ENV=production npm start
+```
+
+The app will run on port 3000 by default and be available at [http://localhost:3000](http://localhost:3000).
+
+### In development mode
+
+Ensure Redis is installed and running.
+
+```
+npm run develop
+```
+
+The app will be running at [http://localhost:3000](http://localhost:3000) or with browsersync at [http://localhost:3001](http://localhost:3001).
+
+#### Running with debugging
+
+Some extra levels of debugging have been added to aid development.
+
+To see all axios requests/responses run:
+
+```
+DEBUG=axios npm run develop
+```
+
+To see app level debug logs run:
+
+```
+DEBUG=app:* npm run develop
+```
+
+You can combine namespaces, for example:
+
+```
+DEBUG=axios,app:* npm run develop
+```
+
+## Components
```

Comment:
Should `Components` and `Book a secure move frameworks` still live in the top level readme as they aren't related to just _development_? 

Could argue `Components` is related to the _development_ of the app but I think it would be good to document the frameworks link at the top level of the repo.

Reply:
Yeah, I think we should have a brief section mentioning both the frameworks and the api at this top level - they're both conceptual dependencies of the app so it's good to surface them a little earlier. If we need in depth details, then that's more likely to end up in a `development` style doc, or in the corresponding repo.

Components I would definitely argue is related to the development of the app.

Reply:
I've added a brief section with the frameworks and the api, linking to both, and linking to the frameworks setup under development.

I think its a nice middle ground between keeping the main Readme simple, and surfacing extra complications earlier.